src/BloomExe/web/ReadersHandler.cs

```diff
@@ -255,6 +258,13 @@ private static void MakeLetterAndWordList(string jsonSettings, string allWords)
 			PathUtilities.OpenFileInApplication(fileName);
 		}
 
+		private static string GetLocalizedString(string key, string defaultText)
+		{
+			return LocalizationManager.UILanguageId == "en"
+				? LocalizationManager.GetDynamicString("Bloom", key, defaultText)
```

Comment:
That code is supporting strings that need localizing in JavaScript. C# things are automatically discovered (L10NSharp uses reflection to find all the calls to GetString). But if I understand correctly, you will actually defeat that by doing this, since the arguments to GetString are not literals. I think it will actually work better as well as being simpler to just call LM.GetString directly in each of the places above.